README.md

```diff
@@ -48,8 +48,16 @@ docker run -t -i -p 26257:26257 cockroachdb/cockroach shell
 # root@82cb657cdc42:/cockroach#
 ```
 
+By default, a range in CockroachDB wants to replicate to a minimum of three nodes. You can run a single node, but you will receive the following error:
```

Comment:
we only start the node in l92, so that paragraph can be there at the earliest. I'm also wondering whether we should use the dev mode for single-node testing runs and have an extra document for production deploys (which basically takes l92 below, with a little more added to get to a three node cluster). The instructions here would then boil down to

```bash
./cockroach start --dev
```

the `./cockroach sql` stuff below would need to be run from the same session or started via `./cockroach sql --dev`.

Whether that's the right thing to do depends on what the typical reader of that section wants - it's certainly the shortest way to interact with the system. What we currently have is "closer" to an actual deployment, but definitely short of it.

I think it's probably a good idea for now - we can link to the "real" way to start a server prominently, and the boilerplate here reduces (and with it, the warning message that prompted this PR).

What do you think? Also ping @tamird.